cli.js

```diff
@@ -60,8 +55,8 @@ const scriptArgs = process.argv.slice(3)
   .join(' ')
 
 searchForScript(__dirname, scriptPath)
-  .then(
-    foundPath => executeScript(foundPath, scriptArgs),
-    triedPaths => throwSearchFail(scriptPath, triedPaths)
-  )
-  .catch(() => process.exit(1))
+  .then(foundPath => executeScript(foundPath, scriptArgs))
+  .catch(e => {
+    console.error(e.toString())
+    process.exit(1)
+  })
```

Comment:
My understanding is that if you had done this, they'd catch:
```js
somePromise
  .then(_ => throw new Error('hey'))
  .then(null, (err) => // catches ^^)
```

But that you can swallow the error in the previous then if you chain another then after not handling the first error:

```js
somePromise
  .then(_ => throw new Error('hey'), err => 'did not handle it!`)
  .catch((err) => // hears nothing... because the fail above swallowed the error)
```

I thought I tested the second erroneous example above and confirmed it before I updated.